_opcodes/op247.html

```diff
@@ -20,3 +20,9 @@
 {% endcapture %}
 
 {% include important.html %}
+
+{% capture note %}
+At least in EEs, the only difference between opcodes #3 and #247, is that #247 will not force the creature(s) to use melee weapons, and is not removed by Opcode #4 (Cure Berserk).
+{% endcapture %}
```

Comment:
The description disagrees: Unlike opcode #3, this one can affect enemies as well.